mapit-gb/settings.py

```diff
@@ -144,12 +144,18 @@
 )
 
 # List of callables that know how to import templates from various sources.
-TEMPLATE_LOADERS = (
-    ('django.template.loaders.cached.Loader', (
+if DEBUG:
+    TEMPLATE_LOADERS = (
```

Comment:
Unlikely to ever matter here, but in general you could set TEMPLATE_LOADERS and then if not DEBUG wrap it in the cached loader (so you don't have to duplicate the lower level loaders).